src/datasets/arrow_dataset.py

```diff
@@ -1449,7 +1450,7 @@ def save_to_disk(
         If you want to store paths or urls, please use the Value("string") type.
 
         Args:
-            dataset_path (`str`):
+            dataset_path (`PathLike`):
```

Comment:
Great, so now this is aligned with the type hint: `dataset_path: PathLike`

Reply:
yes we should have one imo before merging